src/static/css/header.less

```diff
@@ -20,7 +20,7 @@
 			color: @gray-60;
 			.site-org {
 				color: @gray-80;
-				.u-webfont-medium;
+				font-weight: 500;
```

Comment:
Totally unrelated to this PR but what's up with the spacing here?

Reply:
Let's not talk about the quality of the syntax in this repo's Less files. 🙈 🙉 🙊